src/traversals/bipartiteness.jl

```diff
@@ -0,0 +1,48 @@
+"""
```

Comment:
This is bugging me more than I thought. `bipartition` in this round, but I'm ok with further bikeshedding.